.github/workflows/test.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
           - stable-2.13
```

Comment:
please drop 2.13 since it has reached EOL on 06 Nov 2023

Reply:
![image](https://github.com/DelineaXPM/ansible-core-collection/assets/85106843/62148636-7ab5-43b3-b6a6-af5b27b741a4)
link: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix

Reply:
done